extension/src/popup/App.tsx

```diff
@@ -52,6 +57,11 @@ const RouteWrapperEl = styled.div`
   height: 100%;
 `;
 
+// .isBigNumber() not catching correctly, so checking .isBigNumber
+// property as well
+const isSerializable = (value: any) =>
+  value?.isBigNumber || BigNumber.isBigNumber(value) || isPlain(value);
```

Comment:
this was a weird bug. The balances values returned from the wallet sdk fail the .isBigNumber() method test. The BigNumber source code wasn't clear why it fails. I _think_ the source of it comes from `js-stellar-base` is still using [bignumber.js version 4](https://github.com/stellar/js-stellar-base/blob/13e45406f4d67e2ec488c3247bff32f6037b29df/package.json#L107), and isBigNumber method was [added in version 6](https://github.com/MikeMcl/bignumber.js/blob/master/CHANGELOG.md#600).

Regardless, this workaround shouldn't create any issues

Reply:
cc @quietbits for wallet-sdk v2 knowledge

Reply:
Ah weird - good to know though!